TestPullRequest/src/testPackage/TestDataClassRemoveParameter.java

```diff
@@ -26,7 +25,7 @@ public int getLineNumberOfDummyMethod(int a, int b, int c) {
 	}
 
 	public int getLineNumberOfCaller() {
```

Comment:
https://github.com/LHommeDeBat look how cool this is: https://github.com/LHommeDeBat/Refactoring-Bot